lib/scsi_defs.h

```diff
@@ -108,5 +115,29 @@ namespace scsi_defs {
     kCompleted = 0xF
   };
 
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6 Table 58
+  typedef struct InquiryCommand {
+    OptCode optCode = OptCode::kInquiry;
+    bool evpd: 1; // Enable Vital Product Data (EVPD)
+    bool obsolete: 1; // formerly CMDDT
+    uint8_t reserved: 6;
+    uint8_t pageCode: 8;
+    uint16_t allocationLength: 16;
+    uint8_t control: 8;
+    uint16_t padding: 16;
```

Comment:
This seems hacky imo. As far as I can tell, there's nothing in the stl standard that guarantees the internal size of a bitset object. If we want to use absl bit casting, I think using a structure with guaranteed size makes more sense.

Reply:
I agree! I made a new PR to review the memcpy way of constructing the object #24 . Let me know your thoughts on that and we can use that instead